data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2721 @@
+			choice
+				`	"Let me show you my worth in battle. Right now."`
+					goto accept
+				`	(Crush his rodent face, with your fist.)`
+				`	(Run away.)`
+					decline
+			label crush
+			`	You rush his way, fist drawn, and probably give his friends the exact excuse they were looking for to open fire on you. You wake up later, left alone on the floor. Luckily their weapons only stunned you, but you can put a cross on doing any further business with the Unfettered.`
+				decline
+			label accept
+			branch "too many ships"
+				"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
+			`	"Let us prove our worth in battle then, the time for words is over."`
+				accept
+			label "too many ships"
+			`	"Nice try, monkey. First you most park some of your ships! I will not be facing of against a huge fleet. One heavy warship is already almost too much. When that is the case, you will see me in orbit."`
+			``
+			`The Unfettered captain will only show up if you have a small enough amount of ships, that is not currently the case. (To park ships press "I", and then select "Park All" or manually select the ships you want to park. You cannot have more than a heavy warship or a few lighter ships for this fight.)`
+				accept
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target launching
+		ship "Sari'Sea Scorpion (Pulse)" "Intrepid"
+		to spawn
+			"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 12
+		to despawn
+			"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
+		conversation
+			`Now that you have disabled the <npc>, its captain hails you. "It seems you fight well. You have earned my respect today," he says with a voice so calm you are surprised it came from the same captain who was recently provoking you for no apparent reason. In the background you can hear fires roaring. It looks like the damage to his ship was serious.`
+			choice
+				`	"Be careful with provoking me. You may have gotten killed. Do not count on me to repair your ship."`
+					goto dangerous
+				`	"What was this really about?"`
+			`	"You have earned the right to know. We Unfettered always seek to improve ourselves, be it in combat or otherwise. You come from far away, and we do not have many strangers around here. I wanted to find out about new approaches to the fight. It looks like I may have underestimated your ability to fight too... A mistake which I shall not make again."`
```

Comment:
```suggestion
			`	"Ye have earned the right to know. We Unfettered always seek to improve ourselves, be it in combat or otherwise. Ye travel from far away, and we do not see many strangers around here. I wanted to see how ye approached me challenge; I hoped I could find knowledge in how you lost. I may have underestimated yer ability to fight though... A mistake which I shall not make again next time."`
```